packages/workspace/src/core/project-graph/daemon/exec/index.ts

```diff
@@ -0,0 +1,92 @@
+import { logger } from '@nrwl/devkit';
+import { appRootPath } from '@nrwl/tao/src/utils/app-root';
+import { spawn, spawnSync } from 'child_process';
+import { ensureFileSync } from 'fs-extra';
+import { join } from 'path';
+import { dirSync } from 'tmp';
+import {
+  DaemonJson,
+  readDaemonJsonCache,
+  writeDaemonJsonCache,
+} from '../cache';
+import { isServerAvailable } from '../server';
+
+export async function startInBackground(): Promise<void> {
+  /**
+   * For now, while the daemon is an opt-in feature, we will log to stdout when
+   * starting the server, as well as providing a reference to where any subsequent
+   * log files can be found.
+   */
+  const tmpDirPrefix = `nx-daemon--${appRootPath.replace(/\//g, '-')}`;
```

Comment:
That was certainly the thinking, but I'm glad you called it out because I think it's a faulty assumption. The normalization deals with `.` and `..`. The slashes in the normalized path will still be OS dependent.

I have therefore updated it to use the built in `path.sep` to get the character we need to substitute